### PR TITLE
Subtract the sum finished as it is going to get, also changed require…

### PR DESCRIPTION
**Pull request by nxnode**
what in the world! I must have messed something up AGAIN!!
These are not what my files were and now the code in my local is not at all what it was.

### CODE REVIEW

**Comment by nxnode** [comment]
oh wait - i get it! I committed and pushed but main still won't reflect these changes! Haha!! Slowly but surely I'm getting it ;-)

**Comment by W-Ely** [comment]
Looks like, this should not be merged as is :-/

**Comment by nxnode** [comment]
I guess I am still messing something up because even when I change the branch back to NH-Subtract-The-Sums this commit is not showing what I see in Atom

**Review by W-Ely — APPROVED** [review]

File: tests/code_wars/test_subtract_the_sums.py [hunk 1]
```diff
@@ -1,132 +1,143 @@
...
+#     "grape": [86, 84, 82, 61, 40, 38, 19, 17, 15],
+#     "kiwi": [93, 91, 70, 68, 49, 47, 26, 24, 3, 1],
+#     "melon": [97, 95, 76, 74, 53, 51, 32, 30, 28, 7, 5],
+#     "orange": [83, 60, 58, 39, 37, 35, 16, 14],
+#     "pear": [88, 69, 67, 65, 46, 44, 42, 23, 21, 2],
+#     "pineapple": [100, 98, 79, 77, 75, 56, 52, 33, 31, 12, 10, 8],
+# }
+
+
+fruit_dict = [
```
Comment (W-Ely) [inline tests/code_wars/test_subtract_the_sums.py:30]: I missed this I think due to trying to get the actually commits sorted out. 

 - The commented out fruit_dict is no longer relevant to this solution so should be removed.  
 - The `fruit_dict` that is used isn't a dict.  It's pretty common when getting started to name the variables after the Python type,  like book_list or counts_dict.  This really shouldn't be done, if you really need to in order to keep track of what is what, okay, but know that it isn't conventional.   Better names would be book_list --> books and counts_dict --> letter_counts or letter_to_counts. So in this case just `fruits`
 - `fruit_dict` on line 30 is in the `global` scope and is called a `global` variable.  It's available for use anywhere in this file.  Global variable by convention are UPPER_CASE so this should be FRUIT_DICT or rather FRUITS = ..., but anytime a variable is put in the global scope, you have to ask is it needed there.  For example is it used in many functions within the file? Is it used across files? There are probably a few other cases, but in this case no, so it shouldn't be global to begin with but rather housed within the function.

Comment (W-Ely) [reply tests/code_wars/test_subtract_the_sums.py:30]: Create a new clean up branch, and make these fixes and PR for them.

**Pull request merged**